### PR TITLE
Refactor: replace all ocurrences of ESTestCase.getRandom() with LuceneTestCase.random()

### PR DESCRIPTION
**Pull request by camilojd**
Simple refactor.

### CODE REVIEW

File: test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java [hunk 1]
```diff
@@ -208,16 +208,9 @@ public final void ensureCheckIndexPassed() throws Exception {
     // Test facilities and facades for subclasses.
     // -----------------------------------------------------------------
 
-    // TODO: replaces uses of getRandom() with random()
```
Comment (nik9000) [inline test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java]: Huh. I didn't know we had such a TODO.

**Comment by nik9000** [comment]
Looks good to me. I'll see about merging in a few minutes.

**Comment by nik9000** [comment]
Thanks @camilojd ! I had no idea that old TODO was in there and have been using getRandom the whole time. I'm happy to have just one way to do it now.

**Comment by camilojd** [comment]
@nik9000 You're welcome! Glad to help :smiley:

**Pull request merged**